### PR TITLE
docs: add existing landscape research

### PR DESCRIPTION
**Pull request by beeme1mr**
Documented features of the public interfaces for popular feature flag SDKs.

### CODE REVIEW

File: research/exisiting-landscape.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Goal
```
Comment (oleg-nenashev) [inline research/exisiting-landscape.md:1]: Needs title

```suggestion

# Feature Flag Management - API/SDK Landscape Research

## Goal
```

File: research/exisiting-landscape.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+ - ip
+ - country
+ - anonymous
+ - custom
+
+The user interface for NodeJS can be found
+[here](https://github.com/launchdarkly/node-server-sdk/blob/master/index.d.ts#L533).
+
+### Key findings
+ - There's a variationDetails method that returns evaluation and error details.
```
Comment (oleg-nenashev) [inline research/exisiting-landscape.md:163]:
```suggestion
 - There's a `variationDetails` method that returns evaluation and error details.
```

File: research/exisiting-landscape.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
+# Goal
+
+The goal of this research is to document the APIs and SDKs of existing feature flag
+management tools in order to identity patterns that should be considered when
+defining the OpenFeature spec.
+
+> NOTE: This research focuses primarily on the server-side NodeJS SDKs
```
Comment (oleg-nenashev) [inline research/exisiting-landscape.md:7]: Add research status. Is it completed or a living document? How others could contribute? Any feedback needed? Maybe a `## Contribute` section in the beginning would be beneficial

**Review by oleg-nenashev — COMMENTED** [review]
I think it is a nice addition. I suggest also referencing this research from the main README

**Review by AloisReitbauer — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
I will merge it so that we have content in place and work on it incrementally

**Pull request merged**